lib/intro_router.dart

```diff
@@ -1,32 +1,27 @@
 import 'package:flutter/material.dart';
 import 'package:shared_preferences/shared_preferences.dart';
-import 'routes.dart';
-
-class IntroRouter extends StatefulWidget {
-  @override
-  _IntroRouterState createState() => _IntroRouterState();
-}
-
-class _IntroRouterState extends State<IntroRouter> {
-  @override
-  void initState() {
-    super.initState();
-    _checkDisclaimerFirstView();
-  }
+import 'package:wh_covid19/view/disclaimer_view.dart';
```

Comment:
I'm not a flutter dev, but this should fix up the flutter analyze warning

```suggestion
import 'view/disclaimer_view.dart';
```